docs/devel/automation.md

```diff
@@ -68,92 +70,66 @@ The status of the submit-queue is [online.](http://submit-queue.k8s.io/)
 
 ### Ready to merge status
 
+The submit-queue lists what it believes are required on the [merge requirements tab](http://submit-queue.k8s.io/#/info) of the info page. That may be more up to date.
+
 A PR is considered "ready for merging" if it matches the following:
-   * it has the `lgtm` label, and that `lgtm` is newer than the latest commit
-   * it has passed the cla pre-submit and has the `cla:yes` label
-   * it has passed the travis pre-submit tests
-   * one (or all) of
-      * its author is in kubernetes/contrib/submit-queue/whitelist.txt
-      * its author is in contributors.txt via the github API.
-      * the PR has the `ok-to-merge` label
-   * One (or both of)
-      * it has passed the Jenkins e2e test
-      * it has the `e2e-not-required` label
-
-Note that the combined whitelist/committer list is available at
-[submit-queue.k8s.io](http://submit-queue.k8s.io)
+  * The PR must have the label "cla: yes" or "cla: human-approved"
+  * The PR must be mergeable. aka cannot need a rebase
+  * All of the following github statuses must be green
+     * Jenkins GCE Node e2e
```

Comment:
Got it.
Until we have means to not trigger `GCE e2e` and `GCE Node e2e` on PRs that
don't require them, can we have a flag to rerun just the `unit/integration`
tests for certain PRs?
The reason for wanting this split is to merge trivial PRs by reducing the
possibility of test flakes on them.

On Tue, Jun 7, 2016 at 3:49 PM, Eric Paris <notifications@github.com> wrote:

> In docs/devel/automation.md
> <https://github.com/kubernetes/kubernetes/pull/27001#discussion_r66166577>
> :
>
> > -   * it has passed the cla pre-submit and has the `cla:yes` label
> > -   * it has passed the travis pre-submit tests
> > -   * one (or all) of
> > -      * its author is in kubernetes/contrib/submit-queue/whitelist.txt
> > -      * its author is in contributors.txt via the github API.
> > -      * the PR has the `ok-to-merge` label
> > -   * One (or both of)
> > -      * it has passed the Jenkins e2e test
> > -      * it has the `e2e-not-required` label
> > -
> > -Note that the combined whitelist/committer list is available at
> > -[submit-queue.k8s.io](http://submit-queue.k8s.io)
> > +  * The PR must have the label "cla: yes" or "cla: human-approved"
> > +  * The PR must be mergeable. aka cannot need a rebase
> > +  * All of the following github statuses must be green
> > +     * Jenkins GCE Node e2e
>
> This only partly a change what's happening today (which is non-sense)
>
> At this very instant:
> e2e-not-required means that Jenkins GCE e2e need never pass.
> Jenkins GCE Node e2e and Jenkins unit/integration still must pass (at
> least) once.
>
> e2e-not-required cause it to not trigger a retest before merge. But if a
> retest was triggered ALL of e2e, node, and unit/integration had to run (and
> pass) a second time.
>
> This just makes the policy clear. All must pass once. All or none must
> pass twice.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27001/files/2fba6d2d6303d02e919f4e14a24bf7cacc8ca2df#r66166577>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGvIKFbt4sQsv44rwYkq68x0bg_Pvo8fks5qJfV_gaJpZM4IwY5h>
> .
>